src/vector/_methods.py

```diff
@@ -2408,8 +2408,8 @@ def _lib_of(*objects: VectorProtocol) -> Module:  # NumPy-like module
 def _from_signature(
     name: str,
     dispatch_map: typing.Dict[typing.Any, typing.Any],
-    signature: typing.Tuple[typing.Any],
-) -> typing.Tuple[typing.Any]:
+    signature: typing.Tuple[typing.Any, ...],
+) -> typing.Tuple[typing.Any, ...]:
```

Comment:
And this is the signature fix; `Tuple[Any, ...]` is a generic tuple. `Tuple[Any]` is a one-element tuple with anything in it.